components/applications-service/integration_test/service_groups_test.go

```diff
@@ -626,24 +676,32 @@ func habServicesABCD() []*habitat.HealthCheckEvent {
 			withServiceGroup("a.default"),
 			withPackageIdent("core/a/0.1.0/20190101121212"),
 			withHealth("UNKNOWN"),
+			withApplication("a_app"),
+			withEnvironment("a_env"),
```

Comment:
This is the cool part about the new test function to create Hab Event Messages. Just add a single func to update it instead of all the tests and functions and options! cc/ @kmacgugan @danielsdeleo 

![tenor-200387754](https://user-images.githubusercontent.com/5712253/57951016-41e8ce00-78e9-11e9-87a0-d8794852b04f.gif)